app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/ComponentNavigationActivity.java

```diff
@@ -75,6 +77,8 @@
   private static final double DEFAULT_TILT = 0d;
   private static final double DEFAULT_BEARING = 0d;
   private static final int ONE_SECOND_INTERVAL = 1000;
+  private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy_MM_dd_HH_mm_ss");
+  private static final String TXT_EXTENSION = ".txt";
```

Comment:
might want to use `.json` here but not a huge deal

Reply:
Fixing now 🚀